test/quantization/core/test_quantized_module.py

```diff
@@ -1036,6 +1036,33 @@ def test_prelu(self):
         self.assertEqual(qy_ref, qy,
                          msg="PReLU module API failed")
 
+    def test_channel_shuffle(self):
```

Comment:
do you want to add a operator test for torch.channel_shuffle/F.channel_shuffle?

Reply:
I think we already have it: https://github.com/pytorch/pytorch/blob/dcac4dd58edefb6951a60266e53d8767dc9be002/test/quantization/core/test_quantized_op.py#L1298